### PR TITLE
Cluster upgrade orchestration, part 3

### PR DESCRIPTION
**Pull request by Balazs23**
**Description**

Changes proposed in this pull request:

- implement KEB exposes cluster upgrade operation details on /runtimes API

**Related issue(s)**

https://github.tools.sap/kyma/backlog/issues/1165

### CODE REVIEW

File: components/kyma-environment-broker/internal/runtime/converter.go [hunk 1]
```diff
@@ -97,6 +98,20 @@ func (c *converter) ApplyUpgradingKymaOperations(dto *pkg.RuntimeDTO, oprs []int
 	}
 }
 
+// TODO: no totalCount here?
```
Comment (ebensom) [inline components/kyma-environment-broker/internal/runtime/converter.go:101]: Remove TODO

**Review by ebensom — DISMISSED** [review]

File: components/kyma-environment-broker/common/runtime/model.go [hunk 2]
```diff
@@ -76,17 +77,23 @@ type ListParameters struct {
...
 )
 
 func FindLastOperation(rt RuntimeDTO) (Operation, OperationType) {
 	op := *rt.Status.Provisioning
 	opType := Provision
 	// Take the first upgrade operation, assuming that Data is sorted by CreatedAt DESC.
+	if rt.Status.UpgradingCluser.Count > 0 {
+		op = rt.Status.UpgradingCluser.Data[0]
+		opType = UpgradeCluster
+	}
```
[the remark this answers is not in the document]
Comment (Balazs23) [reply components/kyma-environment-broker/common/runtime/model.go:95]: fixed: https://github.com/kyma-project/control-plane/pull/629/commits/da28e074a2b44040f7b538851795fd22cf7ff513

**Review by ebensom — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]
lgtm

**Comment by Balazs23** [comment]
okay, merged changes like branch renaming etc... from main branch to this PR branch

**Pull request merged**